src/cli/commands/generate.ts

```diff
@@ -1,12 +1,25 @@
+import type { CommandDef } from 'citty'
 import * as fsp from 'node:fs/promises'
 import * as path from 'node:path'
 import process from 'node:process'
 import { defineCommand } from 'citty'
 import { consola } from 'consola'
-import { generateDTS } from '../../openapi/generate'
-import { loadConfig } from '../utils'
+import { generateDTS } from '../../openapi/generate.ts'
+import { loadConfig } from '../utils.ts'
 
-export default defineCommand({
+const command: CommandDef<{
```

Comment:
My usecase for TS is to infer as much as possible... This feels overcomplicated.

Reply:
isolatedDeclarations these are mandatory income